### DISCUSSION TITLE
Manpower behind twinBASIC

### DISCUSSION THREAD

**Discussion by uwbwsvd** [twinbasic/twinbasic#339] [General]
I refer to #335 Roadmap.
I wonder about the manpower behind twinBASIC. Manpower of redundant workers. If there is a shortage, twinBasic is skating on thin ice.

**Comment by wqweto** [comment]
TB should outsource stock ActiveX controls like in "Reimplementation of all remaining VB6 shipped controls, such as Winsock, MAPI, Adodc, DataGrid, MSCOMM, etc"

Reimplementing only the DataGrid in C/C++ might be time-sinking and frankly the list is lacking some controls. These controls should be doable in TB although implementing them in VB6 is currently a PITA.

Better enable the core language to be solid and to be able to implement mutli-threading (for MSCOMM) or complex-binding and unsigned data-time or OLEDB interfaces (ADODC/DataGrid).

**Comment by uwbwsvd** [comment]
I hope TB is not a one man show. This could never be successfull on a long shot.

**Reply by WaynePhillipsEA** [reply]
Yes, this development is largely a one-man show, and I've been open about that from the start.  My partner is also involved in the business, and takes care of the admin side of things, which frees up my time as a developer.   As has already been mentioned elsewhere, some tasks will be outsourced to lessen the load, and we are also open to the possibility of partnering with another firm in future if the correct opportunity arises.

This is a big project for sure, but if you've been following the progress so far, you'll have noticed that we've already come a very, very long way.   We've now got a very strong and capable compiler that will be the root of everything else to come.   

I suspect many people would not had envisaged that a one-man show could even have got as far as we are today with tB... and yet here we are.  As the old saying goes, 'the proof is in the pudding'... so stick around to find out what's for dessert.

**Reply by Kr00l** [reply]
> I hope TB is not a one man show. This could never be successfull on a long shot.

Being a one man show have also some pro's. He is boss, project manager and developer in one person. So the meetings are quite efficient. :>
